### PR TITLE
changelog: Update changelog with Bluetooth RPC changes

### PR DESCRIPTION
**Pull request by KAGA164**
Updates changelog with changes in the Bluetooth
Host serialization like: support for the GATT
server API serialization and Kconfig configuration
which enables Bluetooth RPC

### CODE REVIEW

**Review by grochu — APPROVED** [review]
Looks good

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -221,6 +224,14 @@ Libraries
 
 This section provides detailed lists of changes by :ref:`library <libraries>`.
 
+Bluetooth libraries
+-------------------
+
+Updated:
+
+* Added support for the GATT Server API serialization in the :ref:`ble_rpc` library.
+* Changed the config which enables the :ref:`ble_rpc` library from the :kconfig:`CONFIG_BT_RPC` to the :kconfig:`CONFIG_BT_RPC_STACK`.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:233]:
```suggestion
* :ref:`ble_rpc` library:

  * Added support for the GATT Server API serialization.
  * Changed the configuration option that enables the library from the :kconfig:`CONFIG_BT_RPC` to the :kconfig:`CONFIG_BT_RPC_STACK`.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -125,6 +125,9 @@ Bluetooth samples
 * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
 * :ref:`ble_llpm` sample - Added role selection.
   The user now selects the role for each board by typing "m" or "s" in the terminal emulator.
+* Changed the default sample for the :ref:`ble_rpc` to the :ref:`peripheral_uart` sample. This sample runs out of the box with a serialized Bluetooth Low Energy Host.
+* Added the :ref:`ble_rpc_host:` child image configuration overlay for the :ref:`peripheral_hids_mouse` sample as an example how to configure application running a serialized Bluetooth Low Energy Host.
+  The :ref:`peripheral_hids_mouse` runs out the box with the :ref:`ble_rpc` library.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:130]:
```suggestion
* :ref:`peripheral_uart` sample is now the default sample for the :ref:`ble_rpc` library.
  The sample runs out of the box with a serialized Bluetooth Low Energy Host.
* :ref:`peripheral_hids_mouse` sample now comes with the :ref:`ble_rpc_host` child image configuration overlay.
  The overlay shows how to configure application running a serialized Bluetooth Low Energy Host.
  The :ref:`peripheral_hids_mouse` runs out the box with the :ref:`ble_rpc` library.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 2 above)
Comment (peknis) [reply doc/nrf/releases/release-notes-changelog.rst:130]: The overlay shows how to configure **an** application...

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 1 above)
Comment (KAGA164) [reply doc/nrf/releases/release-notes-changelog.rst]: Done

File: doc/nrf/releases/release-notes-changelog.rst (hunk 2 above)
Comment (KAGA164) [reply doc/nrf/releases/release-notes-changelog.rst]: Done

**Review by peknis — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Pull request merged**